Source/Urho3D/Engine/Engine.cpp

```diff
@@ -63,8 +63,25 @@
 #include "../Urho2D/Urho2D.h"
 #endif
 
-#if defined(__EMSCRIPTEN__) && defined(URHO3D_TESTING)
+#ifdef __EMSCRIPTEN__
+#include "../Graphics/GraphicsEvents.h"
 #include <emscripten/emscripten.h>
+#include <emscripten/bind.h>
+static const Urho3D::Context *appContext;
+
+static void JSCanvasSize(int width, int height, bool fullscreen, float scale)
+{
+    using namespace Urho3D;
+    URHO3D_LOGINFOF("JSCanvasSize: %d x %d", width, height);
+    appContext->GetSubsystem<Graphics>()->SetMode(width, height);
+    UI* ui = appContext->GetSubsystem<UI>();
+    ui->SetScale(scale);
+}
+
+using namespace emscripten;
+EMSCRIPTEN_BINDINGS(Module) {

```

Comment:
Brace style

Reply:
Seems like this macro doesn't like a case without braces